packages/mrm-core/src/npm.js

```diff
@@ -267,6 +268,14 @@ function getUnsatisfiedDeps(deps, versions, options) {
 	});
 }
 
+function init() {
```

Comment:
Is this `rc` library the same as npm is using internally? The question's how well it matches the npm config search behavior?

(I was also thinking about using cosmiconfig, which seems to be the most popular too for the task, to load Mrm's own config files, and would rather avoid having two libraries for the same task, unless there are good reasons.)

Reply:
Was just using the library in your `user-meta` project. End of my day here, but for npm's own, it seems maybe https://github.com/npm/config handles their config.

Btw, I assume you are following the practice of taking pulls against `next`, so rebasing now against that rather than `master`.

Reply:
Fine then — nobody complained about it so far ;-)

`next` is specifically for the next **major** to combine all the breaking changes together. So if you want to have this pull request release soon, better keep using `master`.